src/java.management.rmi/share/classes/javax/management/remote/rmi/RMIConnectorServer.java

```diff
@@ -149,7 +149,7 @@
      * classes they use in their serial form.
      * <p>
      * Care must be taken when defining such a filter, as defining
-     * a white list too restrictive or a too wide a black list may
+     * an accept-list too restrictive or a too-wide reject-list may
```

Comment:
would "an allow-list too restrictive or a reject-list too wide" read better?

Reply:
I agree that there is room for improvement here.  How about:
"...an allow-list too restrictively, or a reject-list too broadly, may..."
?

Reply:
That sounds good to me

Reply:
It's an adverb, since it's the act of 'defining' that is being done too restrictively or broadly. If you want to have an adjective you would need to rephrase it so it applies to the noun, like 'defining a too restrictive accept-list'. My non-native English vibes would still prefer the former.

Reply:
I'd suggest `... as defining an allow-list that is too narrow or a reject-list that is too-wide may prevent ...`.

Reply:
Even better!